### DISCUSSION TITLE
Hello!

### DISCUSSION THREAD

**Discussion by kil0byt3** [Rebirth-of-the-Night/Rebirth-Of-The-Night#910] [General]
he do be vibin

**Comment by democat3457** [comment]
doo be doo be do

**Reply by kil0byt3** [reply]
> doo be doo be do

yes

**Comment by kil0byt3** [comment]
new thread

**Reply by democat3457** [reply]
ok can you actually put one into the Q&A section

**Reply by kil0byt3** [reply]
oh i thought you wanted one NOT in the Q&A section lul

**Reply by democat3457** [reply]
i did, but it turns out i cant mark things as an answer in a non-Q&A section
